src/tests/ftest/server/metadata.py

```diff
@@ -103,124 +122,60 @@ def thread_control(self, threads, operation):
-                # a few container creates to succeed in the interim.
-                self.log.info("Phase 3: sustained container creates: "
-                              "to nospace and beyond")
-                big_array = []
-                in_failure = False
-                for _cont in range(30000):
-                    try:
-                        container = DaosContainer(self.context)
-                        container.create(self.pool.pool.handle)
-                        big_array.append(container)
-                        if in_failure:
-                            self.log.info("Phase 3: nospace -> available "
-                                          "transition, cont %d", _cont)
-                            in_failure = False
-                    except DaosApiError:
-                        if not in_failure:
-                            self.log.info("Phase 3: available -> nospace "
-                                          "transition, cont %d", _cont)
-                        in_failure = True
-
-                self.log.info("Phase 3: passed (created %d / %d containers)",
-                              len(big_array), 30000)
-                return
-
-            except DaosApiError as exe2:
-                print(exe2, traceback.format_exc())
-                self.fail("Phase 2: fail (unexpected container destroy error)")
-
-        # Phase 1 failure
-        self.fail("Phase 1: failed (metadata full error did not occur)")
-
-    @avocado.fail_on(DaosApiError)
-    def test_metadata_addremove(self):
-        """JIRA ID: DAOS-1512.
+        return container_array, container_length
 
-        Test Description:
-            Verify metadata release the space after container delete.
+    def delete_containers_in_array(self, container_array):
+        """Creates containers in pool
 
-        Use Cases:
-            ?
+        Args:
+            container_array (List): List of DaosContainer instances
 
-        :avocado: tags=metadata,metadata_free_space,nvme,large,hw
-        :avocado: tags=full_regression
+        Returns:
+            list: If all containers are destroyed correctly, an empty list
+                  otherwise an exception is raised
+        Raises:
+            DaosApiError
         """
-        self.pool.pool.connect(2)
-        for k in range(10):
-            container_array = []
-            self.log.info("Container Create Iteration %d / 9", k)
-            for cont in range(NO_OF_MAX_CONTAINER):
-                container = DaosContainer(self.context)
-                try:
-                    container.create(self.pool.pool.handle)
-                except DaosApiError as exc:
-                    self.log.info("Container create %d/%d failed: %s",
-                                  cont, NO_OF_MAX_CONTAINER, exc)
-                    self.fail("Container create failed")
+        self.log.info("Cleaning up containers after ")
+        for cont in range(len(container_array)):
```

Comment:
(pylint-consider-using-enumerate) Consider using enumerate instead of iterating with range and len